### PR TITLE
codeofconduct

### PR DESCRIPTION
**Pull request by prayagb**
**Add a code of conduct page #59**

Copied text from github and created 2 versions - one with text-column to be in sync with other pages and one without text-column. Let me know which is preferable along with any other suggestions..
code-1
code-2
code-3
code-4
code-5
code-6

### CODE REVIEW

**Comment by borisrorsvort** [comment]
@prayagb  I'd keep the first version (with 2 column) but now that I'm thinking about it the second part would fit more in a about page. So you can remove it from here but I'll keep it for inspiration when I'll redo the about page.

**Comment by prayagb** [comment]
Removed the one-column version and retained the 2-column version

File: source/code-of-conduct/index.haml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      %h3.header__subtitle.red
+        Our contact information
+      .container.black
+      .text-columns
+        If you have any questions don't hesitate to get in touch with board members or activities staff. An activity staff is available both during, before and after the activity. You can reach activity staff by the means they have communicated for the activity. You can reach board members by
+        %span
+          = link_to 'email', 'mailto: info@rubybelgium.org'
+        %span or by
+        %span
+          = link_to 'twitter.', 'https://twitter.com/brug_be'
```
Comment (toch) [inline source/code-of-conduct/index.haml]: it is https://twitter.com/rubybelgium now

File: source/code-of-conduct/index.haml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+        %p Associate and full members, volunteers, and activity participatns are subject to this policy at all association activities and associate-related social events. We invite all our members and those who participate in our activities to help us create safe and positive experiences for everyone.
+
+      %h3.header__subtitle.red
+        Our contact information
+      .container.black
+      .text-columns
+        If you have any questions don't hesitate to get in touch with board members or activities staff. An activity staff is available both during, before and after the activity. You can reach activity staff by the means they have communicated for the activity. You can reach board members by
+        %span
+          = link_to 'email', 'mailto: info@rubybelgium.org'
```
Comment (toch) [inline source/code-of-conduct/index.haml]: please update the email to safe@rubybelgium.be

**Comment by toch** [comment]
Thanks a lot @prayagb there are two comments to address.

Could you also add a link to the GitHub repository of the code of conduct at then end. Something link "would you like to suggest improvements, or corrections to the present code of conduct, please submit them to its repository."

**Comment by prayagb** [comment]
Changed email to safe@rubybelgium.be and added link to the github repo.

File: source/code-of-conduct/index.haml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+      %h3.header__subtitle.red
+        As activity staff or volunteer
+      .container.black
+      .text-columns
+        %p Ruby Belgium members and volunteers should be aware of their influence and realize what they do and say reflects on the association in general. You should act as an exemplary role. We do not tolerate any kind of harassment in a activity or a talk. Please keep in mind that a harmless joke to you might offend another person.
+
+      .extraline
+      .container.text-center.red
+        If would you like to suggest improvements or corrections to the present code of conduct, please submit them to its Github
+        %span.underline
```
Comment (borisrorsvort) [inline source/code-of-conduct/index.haml]: feels weird to have this underline class for a link. Can you remove it ?

**Comment by prayagb** [comment]
sure. should I otherwise leave it as it is (only the word 'repository' is a link) or should I make the whole line a link - the reason I am asking is that it is not evident that the word 'repository' is a link....

**Comment by borisrorsvort** [comment]
Links are red by default so actually remove the .red class. Keep the text in black and the link would be red automatically.
Also you should rebase on master

**Comment by prayagb** [comment]
Ok rebased master and removed underline

File: source/code-of-conduct/index.haml [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+        %p Actitivity staff and board members are happy to help participants contact local law enforcement, provide escorts, or in any other way assist those experiencing harassment. Having a safe environment for our members and activity participants is of paramount importance to us, and we value your help towards creating this space.
+
+        %p Beyond creating a safe environment, we want to encourage open citizenship. Communities mirror the societies in which they exist and positive action is essential to counteract the many forms of inequality and abuses of power that exist in society. If you see someone who is making an extra effort to ensure our community is welcoming, friendly, and encourages all participants to contribute to the fullest extent, we want to know.
+
+        %p Associate and full members, volunteers, and activity participatns are subject to this policy at all association activities and associate-related social events. We invite all our members and those who participate in our activities to help us create safe and positive experiences for everyone.
+
+    %h3.header__subtitle.red.left-spaced
```
Comment (borisrorsvort) [inline source/code-of-conduct/index.haml]: For sub header keep the consistency with http://rubybelgium.be/rails-girls/ and remove .left-spaced

File: source/code-of-conduct/index.haml [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+        If you have any questions don't hesitate to get in touch with board members or activities staff. An activity staff is available both during, before and after the activity. You can reach activity staff by the means they have communicated for the activity. You can reach board members by
+        %span
+          = link_to 'email', 'mailto: safe@rubybelgium.org'
+        %span or by
+        %span
+          = link_to 'twitter.', 'https://twitter.com/rubybelgium'
+        %p
+          %p As reminder, the emergency number in Belgium is the 112.
+
+    %h3.header__subtitle.red.left-spaced
```
Comment (borisrorsvort) [inline source/code-of-conduct/index.haml]: Same here

**Comment by prayagb** [comment]
removed left-spaced class but the subheadings have turned black. not easily turning red. should I continue to try and change to red? if you have a quick tip, I would love to hear....

**Comment by borisrorsvort** [comment]
If you look in other pages, titles who are in a .header are red. Thar's one way ;) Make sure they centered also

**Comment by prayagb** [comment]
maybe finally got it?
Thanks for your patience

**Comment by borisrorsvort** [comment]
Dont reivent the wheel, try to reuse what on other pages ;) 
developer_tools_-_httprubybelgium berails-girls_2016-07-10_10-13-26

**Comment by prayagb** [comment]
does it look ok now? it certainly looks better to me :smile:

**Comment by prayagb** [comment]
hahaha, this one took a while - but a couple of lessons learnt....

**Comment by borisrorsvort** [comment]
@prayagb I had to make some adjustments, have a look https://github.com/brug-be/rubybelgium/commit/c73c3482995cee1fb822ca837f9b6aa520966177

**Comment by toch** [comment]
Great job :clap:

**Pull request merged**